lua/lazyvim/plugins/extras/overseer/core.lua

```diff
@@ -0,0 +1,73 @@
+return {
+  {
+    "catppuccin",
+    optional = true,
+    opts = {
+      integrations = { overseer = true },
+    },
+  },
+  {
+    "folke/which-key.nvim",
+    optional = true,
+    opts = {
+      defaults = {
+        ["<leader>o"] = { name = "+overseer" },
+      },
+    },
+  },
+  {
+    "stevearc/overseer.nvim",
+    cmd = {
+      "OverseerOpen",
+      "OverseerClose",
+      "OverseerToggle",
+      "OverseerSaveBundle",
+      "OverseerLoadBundle",
+      "OverseerDeleteBundle",
+      "OverseerRunCmd",
+      "OverseerRun",
+      "OverseerInfo",
+      "OverseerBuild",
+      "OverseerQuickAction",
+      "OverseerTaskAction",
+      "OverseerClearCache",
+    },
+    dependencies = "nvim-telescope/telescope.nvim",
```

Comment:
I don't believe this should be a hard dependency?

Reply:
You are right